static/css/main.css

```diff
@@ -181,6 +181,12 @@ th.goog-tablesorter-sorted-reverse .sorter_icon:after {
     width: 80%;
 }
 
+// TODO(Peter): This should really be 16 bytes, compared to 4 for ipv4
```

Comment:
I guess this file is not preprocessed since it's in the `static` folder? Then you should use `/* … */` for comments, CSS doesn't support `//`.

Reply:
Good catch @FloEdelmann . I've also raised #266 to add a CSS linter to catch such stupidity in future. Can you recommend one?

Reply:
Unfortunately, I can't as I have never used one in production. [Stylelint](https://stylelint.io/) looks good though and would probably be my first choice.